__tests__/app.test.js

```diff
@@ -119,3 +119,67 @@ describe('"GET /api/articles', () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Returns an array containing all comments for specified article", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toEqual([
```

Comment:
is `.toEqual` the most appropriate Jest matcher? What if you want to expand on functionality later? Will you go back and change all of your tests?